src/MoBi.Core/Services/QuantityTask.cs

```diff
@@ -213,22 +211,17 @@ private ICommand setParameterDefaultStateInSimulation(IParameter parameter, bool
          return new SetParameterDefaultStateInSimulationCommand(parameter, defaultState, simulation).Run(_context);
       }
 
-      private ICommand synchronizedCommand(IQuantity quantity, IMoBiSimulation simulation, IMoBiCommand simulationCommandToBeRun, bool shouldUpdateValueOriginAndState = true)
+      private ICommand runAsSimulationMacroCommand(IQuantity quantity, IMoBiSimulation simulation, IMoBiCommand simulationCommandToBeRun, bool shouldUpdateValueOriginAndState = true)
       {
          var macroCommand = new MoBiMacroCommand();
 
-         //add one before setting the value in the simulation to enable correct undo
-         macroCommand.Add(_quantitySynchronizer.Synchronize(quantity, simulation));
-
          IOSPSuiteCommand executedCommand = simulationCommandToBeRun.AsHidden().Run(_context);
 
          if (shouldUpdateValueOriginAndState)
             executedCommand = withUpdatedDefaultStateAndValueOrigin(executedCommand, quantity, simulation);
 
          macroCommand.Add(executedCommand);
```

Comment:
I don't really get why we have a macro command now. It looks like we have one command that we keep updating but I am not sure. For me, the macro command is just complexity for nothing

Reply:
removed it.